tests/amqp_pause_test.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash -x
+# Test for amqp queue listening to be paused in a paused producer
+
+echo 'making sure we are clean.  one or both of these commands may fail.'
+killall nodeos
+rm -rf var/lib/node_004
```

Comment:
Seems fragile to reach in and remove this directory. The python scripts have an option to leave directories in place, this would violate that.